app/handlers/sessions_trusted_launch.rb

```diff
@@ -0,0 +1,32 @@
+class SessionsTrustedLaunch
+
+  lev_handler
+
+  protected
+
+  def authorized?
+    true
+  end
+
+  def handle
+    signup_state = options[:signup_state]
+    uuid_link = UserExternalUuid.find_by_uuid(signup_state.trusted_data['uuid'])
+    if uuid_link.present?
+      outputs[:user] = uuid_link.user
+      signup_state.sign_in!(outputs[:user])
```

Comment:
It is late, so I may be off my rocker .... but `signup_state` is an instance of `SignupState`, right?  It doesn't have a `sign_in!` method that I see?

Reply:
yeah - that's a bug :(  Should be calling the method on the controller reference.  All OOB now of course.